src/cljs/nnts2/events.cljs

```diff
@@ -4,8 +4,8 @@
    [day8.re-frame.http-fx]
    [nnts2.db :as db]
    [nnts2.note.events :as note-events]
-   [nnts2.organization.api :as org-api]
-   [nnts2.user.api :as user-api]))
+   [nnts2.organization.events :as org-events]
```

Comment:
Unused imports?

Reply:
no you have to include all events here, even if they dont seem to be used in the code, otherwise they are just not dispatched.